### PR TITLE
Adding a settings.yml

### PR DESCRIPTION
**Pull request by bpqoc**
## Task

Updating the workflow for our repositories

## Description

Creating the `settings.yml` so that the standard rules we have for our repos are defaulted in.

**Note**: requires that the settings app for github is configured for the organization

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  - name: UAT
+    protection:
+      required_pull_request_reviews:
+        required_approving_review_count: 0
+        dismiss_stale_reviews: true
+      required_status_checks: null
+      enforce_admins: null
+      required_linear_history: false
+      restrictions: null
+  # no required revierwers, requires linear history to force squash merging
```
Comment (bpqoc) [inline .github/settings.yml:56]:
```suggestion
  # no required reviewers, requires linear history to force squash merging
```

**Comment by bpqoc** [comment]
Tried it in https://github.com/QOC-Innovations/testerino

**Comment by bpqoc** [comment]
> Tried it in https://github.com/QOC-Innovations/testerino and a personal repo.
> 
> I'm not sure if rulesets get applied across across repos from .github though since they are generally disabled unless using a paid version. If it doesn't we need .settings.yml on all repos or to do it via the rest api.

Just checked in personal repos, settings will apply to newly created repos and if the settings.yml in this repo is updated again those changes won't apply to existing repos.

**Review by TFqoc — APPROVED** [review]
🚀

**Pull request merged**